filament/backend/src/vulkan/VulkanFboCache.cpp

```diff
@@ -104,26 +111,46 @@ VkRenderPass VulkanFboCache::getRenderPass(RenderPassKey config) noexcept {
         .pDepthStencilAttachment = hasDepth ? &depthAttachmentRef : nullptr
     };
 
-    // The attachment description specifies the layout to transition to at the END of the render pass.
+    // Set up some const aliases for terseness.
+    const VkAttachmentLoadOp clear = VK_ATTACHMENT_LOAD_OP_CLEAR;
+    const VkAttachmentLoadOp dontCare = VK_ATTACHMENT_LOAD_OP_DONT_CARE;
+
+    // NOTE: We do not support LOAD_OP_LOAD for the swap chain. This is rarely what you want anyway,
```

Comment:
Wouldn't that break when rendering several Views?

Reply:
That's an excellent point, multiple views seem to work on Molten but I bet they would break on Android for this reason.

For now, let me change the NOTE into TODO, then work on this after merging.

Reply:
And we have a nice test for this, btw.